### ISSUE TITLE
`make` fails with new `expected` tests

### ISSUE THREAD

**Issue by Sir4ur0n** [c-cube/qcheck#145]
Today I pulled `master` and now `make` fails as such:

```shell
$ make
Done: 0/0 (jobs: 0)File "example/alcotest/output.txt.expected", line 1, characters 0-0:
diff --git a/example/alcotest/output.txt.expected b/example/alcotest/output.txt
index 0f5bdb3..57465fb 100644
--- a/example/alcotest/output.txt.expected
+++ b/example/alcotest/output.txt
@@ -15,8 +15,8 @@ test `fail_sort_id` failed on ≥ 1 cases: [1; 0] (after 20 shrink steps)
 [exception] test `fail_sort_id` failed on ≥ 1 cases: [1; 0] (after 20 shrink steps)

             Raised at file "src/core/QCheck2.ml", line 1759, characters 6-38
-            Called from file "src/alcotest-engine/core.ml", line 229, characters 17-23
-            Called from file "src/alcotest-engine/monad.ml", line 24, characters 31-35
+            Called from file "src/alcotest-engine/core.ml", line 277, characters 17-23
+            Called from file "src/alcotest-engine/monad.ml", line 34, characters 31-35


  ──────────────────────────────────────────────────────────────────────────────
         git (internal) (exit 1)
(cd _build/default && /nix/store/pj43dfd5k4zb0ivafkyqpr1blzd616ry-git-2.27.0/bin/git --no-pager diff --no-index --color=always -u example/alcotest/output.txt.expected example/alcotest/output.txt)
Done: 90/94 (jobs: 1)Testing `QCheck'.
This run has ID `8BF9AADF-F5CD-4143-8EF0-702F59E8C77C'.

  [OK]          Shrink          0   int_towards.
  [OK]          Shrink          1   int32_towards.
  [OK]          Shrink          2   int64_towards.
  [OK]          Shrink          3   float_towards.
  [OK]          Gen             0   opt with default ratio.
  [OK]          Gen             1   opt with custom ratio.
  [OK]          Test            0   make with custom count.
  [OK]          Test            1   make with custom count.
  [OK]          Test            2   make with default count.
  [OK]          Test            3   make with env count.
  [OK]          String          0   shrinking.

Full test results in `~/code/qcheck/_build/default/test/core/_build/_tests/QCheck'.
Test Successful in 0.056s. 11 tests run.
make: *** [Makefile:8: test] Error 1
```

This looks like an Alcotest version difference, but how is one supposed to know the right version to pick? I think we lack reproducibility, given that the "expected" test is now pretty sensitive...

**Comment by c-cube** [comment]
My bad. There is provision for that in other tests, look for grep -v to filter these lines.

**Comment by c-cube** [comment]
please check again :)

**Comment by Sir4ur0n** [comment]
Fixed by 18247cf40af4272f7a2f93e273724b962db61b01 👍 
Thank you!
